Unable to import module 'src\\email-password\\signup-lambda' on Windows

```
"logs": [
    {
      "START RequestId: a95a2f2f-": "b1c6-11e7-af17-e1bfe467d931 Version: $LATEST\nUnable to import module 'src\\email-password\\signup-lambda': Error\n    at Function.Module._resolveFilename (module.js:469:15)\n    at Function.Module._load (module.js:417:25)\n    at Module.require (module.js:497:17)\n    at require (internal/module.js:20:19)\nEND RequestId: a95a2f2f-b1c6-11e7-af17-e1bfe467d931\nREPORT RequestId: a95a2f2f-b1c6-11e7-af17-e1bfe467d931\tDuration: 51.09 ms\tBilled Duration: 100 ms \tMemory Size: 512 MB\tMax Memory Used: 19 MB\t\n"
    }
  ],
  "returnValue": {
    "errorMessage": "Cannot find module '/var/task/src\\email-password\\signup-lambda'",
    "errorType": "Error",
    "stackTrace": [
      "Function.Module._load (module.js:417:25)",
      "Module.require (module.js:497:17)",
      "require (internal/module.js:20:19)"
    ]
  }
```

Getting a similar problem with 0.8.0-alpha.5. Running `graphcool init` and `graphcool deploy` then trying to query the hello function. Running Windows 10 with Node 8.4.0. 
Log:
```
  "logs": [
    "START RequestId: 03471105-b1dc-11e7-8012-b7796e737c07 Version: $LATEST\nUnable to import module 'src\\hello-lambda': Error\n    at Function.Module._load (module.js:417:25)\n    at Module.require (module.js:497:17)\n    at require (internal\/module.js:20:19)\nEND RequestId: 03471105-b1dc-11e7-8012-b7796e737c07\nREPORT RequestId: 03471105-b1dc-11e7-8012-b7796e737c07\tDuration: 1.23 ms\tBilled Duration: 100 ms \tMemory Size: 512 MB\tMax Memory Used: 20 MB\t\n"
  ],
  "returnValue": {
    "errorMessage": "Cannot find module '\/var\/task\/src\\hello-lambda'",
    "errorType": "Error",
    "stackTrace": [
      "Module.require (module.js:497:17)",
      "require (internal\/module.js:20:19)"
    ]
  }
```

--- a/cli/packages/graphcool-cli-core/src/commands/deploy/Bundler/Bundler.ts
+++ b/cli/packages/graphcool-cli-core/src/commands/deploy/Bundler/Bundler.ts
@@ -186,7 +186,7 @@ export default class Bundler {
         typeof fn.fn.handler.code === 'string'
           ? fn.fn.handler.code
           : fn.fn.handler.code!.src
-      const buildFileName = path.join(this.buildDir, this.getBuildFileName(src))
+      const buildFileName = path.posix.join(this.buildDir, this.getBuildFileName(src))
       const lambdaHandlerPath = this.getLambdaHandlerPath(buildFileName)
       const devHandlerPath = this.getDevHandlerPath(buildFileName)
       const bylinePath = this.getBylinePath(buildFileName)
@@ -220,7 +220,7 @@ export default class Bundler {
     )
   }
 
-  getBuildFileName = (src: string) => path.join(src.replace(/\.ts$/, '.js'))
+  getBuildFileName = (src: string) => path.posix.join(src.replace(/\.ts$/, '.js'))
   getLambdaHandlerPath = (fileName: string) =>
     fileName.slice(0, fileName.length - 3) + '-lambda.js'
   getLambdaHandler = (fileName: string) =>

